keylime/cli/options.py

```diff
@@ -9,9 +9,7 @@ class UserError(Exception):
     pass
 
 
-def get_opts_error(args) -> Tuple[bool, str]:
-    if args.ima_exclude and not args.allowlist:
-        return True, "--exclude cannot be used without an --allowlist"
+def get_opts_error(args):
```

Comment:
Is OK to remove the type annotation? I think there is an ongoing PR that is enforcing this.

Reply:
Oops, I think that may have been something gone wrong with my rebase...